### PR TITLE
Initial rpc rollout.

### PR DESCRIPTION
**Pull request by hellysmile**

### CODE REVIEW

**Comment by hellysmile** [comment]
@alternativshik hey, is there any chance that You can take a look on refactoring of rabbitmq bindings?

**Review by alternativshik — APPROVED** [review]
Producer code looks good. Should I check that it works properly?

**Comment by hellysmile** [comment]
> Producer code looks good. Should I check that it works properly?

That would be awesome, i am not so into queue bindings

Not sure how to test it properly

Thanks a lot for your time

**Comment by alternativshik** [comment]
> > Producer code looks good. Should I check that it works properly?
> 
> That would be awesome, i am not so into queue bindings
> 
> Not sure how to test it properly
> 
> Thanks a lot for your time

I'm working on a new code right now and will use rabbitmq, so I will test it in a week or so.

**Review by froosty — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -16,31 +16,30 @@ Installation
 
     pip install aioamqp_consumer
 
-Usage
------
+Consume/Producer usage
```
Comment (hzlmn) [inline README.rst]: ConsumeR :)

**Review by hzlmn — APPROVED** [review]

**Comment by alternativshik** [comment]
@hellysmile It's OK

**Pull request merged**